.gitignore

```diff
@@ -4,5 +4,5 @@
 *.a
 *.pyc
 __pycache__
-sktools/build
-
+*build/
+*_build/
```

Comment:
I would still limit this to directories in the repository root only, *i.e.* leading `/`.